src/systems/performer_detector/PerformerDetector.hh

```diff
@@ -67,6 +67,8 @@ namespace systems
   /// `<geometry>`: Detection region. Currently, only the `<box>` geometry is
   /// supported. The position of the geometry is derived from the pose of the
   /// containing model.
+  /// `<pose>`: Additional pose offset. This pose is added to the parent
+  /// model pose when computing the detection region.
```

Comment:
mention that the rotational component of pose is not used?

Reply:
Added: 359759f00c2ce098250f14bd032c7acc37398d32